### DISCUSSION TITLE
Chat responses are incomplete

### DISCUSSION THREAD

**Discussion by jneutron81** [open-webui/open-webui#2994] [General]
reluctantly asking for assistance, spent a few days eliminating possibilities and im stuck. 

Open webui stops displaying the chat reply when using larger models (llama3 70b) and watching cpu & gpu usage i can see that ollama is still outputting a response. 

- verified ollama completes responses at the command line
- using smaller models (llama3 7b) responses are completed
- refreshing the webpage the entire chat response is lost
- have received chrome notifications of not responding, close/wait
- tested in firefox and mobile browsers with same results
- tested both cuda and non cuda versions of open-webui, running in docker

Open webui was running great priori to updating from v0.1.125 to v0.2.0 and this challenge persists in v0.3.1 and i have reinstalled ubuntu to verify there was no system issue.

attached screenshot for reference, any help is apprecated.

![image](https://github.com/open-webui/open-webui/assets/172321015/624aedfe-86eb-4383-800e-6bcbb0515a30)

**Comment by jneutron81** [comment]
watching console in Chrome

![image](https://github.com/open-webui/open-webui/assets/172321015/7b0a2796-e4ad-4850-87a3-c67a329aeea1)

**Comment by R-Bower** [comment]
+1, I'm also looking for a fix here.

**Comment by rgaricano** [comment]
are to tried increasing max_tokens &| num_ctx ?

Another reason could be that with large models, when the context increases, it exceeds the GPU's VRAM capacity and switches to the CPU, shortening the response. (due to the switch itself or by some timeout)
